slam-ws/src/phidgets_drivers/imu_filter_madgwick/cfg/ImuFilterMadgwick.cfg

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
+# IMU Filter dynamic reconfigure
+
+PACKAGE='imu_filter_madgwick'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+                                                                    
+gen.add("gain", double_t, 0, "Gain of the filter. Higher values lead to faster convergence but more noise. Lower values lead to slower convergence but smoother signal.", 0.1, 0.0, 1.0) 
+gen.add("zeta", double_t, 0, "Gyro drift gain (approx. rad/s).", 0, -1.0, 1.0) 
+gen.add("mag_bias_x", double_t, 0, "Magnetometer bias (hard iron correction), x component.", 0, -10.0, 10.0)
```

Comment:
add the -10 degree decline here?

Reply:
No, the -10 degree decline is called magnetic declination and not magnetic bias, I already accounted for the bias in the .yaml file

Reply:
Also, magnetic declination is added in `navsat_transform_node`, which is not in `dev` yet

Reply:
ah okok then looks good to me